Makefile

```diff
@@ -105,6 +105,27 @@ verify-proto-annotations:
 verify-genproto:
 	PASSES="genproto" ./scripts/test.sh
 
+# Failpoints
+
+.PHONY: gofail-enable
+gofail-enable: install-gofail
+	gofail enable server/etcdserver/ server/storage/backend/ server/storage/mvcc/
+	cd ./server && go get go.etcd.io/gofail/runtime@cb2ba30a3077
+	cd ./etcdutl && go get go.etcd.io/gofail/runtime@cb2ba30a3077
+	cd ./etcdctl && go get go.etcd.io/gofail/runtime@cb2ba30a3077
+	cd ./tests && go get go.etcd.io/gofail/runtime@cb2ba30a3077
+
+.PHONY: gofail-disable
+gofail-disable:
+	gofail disable server/etcdserver/ server/storage/backend/ server/storage/mvcc/
+	cd ./server && go mod tidy
+	cd ./etcdutl && go mod tidy
+	cd ./etcdctl && go mod tidy
+	cd ./tests && go mod tidy
+
+.PHONY: install-gofail
+install-gofail:
+	cd tools/mod; go install go.etcd.io/gofail@cb2ba30a3077
```

Comment:
Can you use the latest commit ID fc21f61ba88a639ebd199a33bbb298093b21b0ed?

Reply:
Sure